gapic/src/main/com/google/gapid/models/Devices.java

```diff
@@ -246,6 +247,15 @@ public void removeListener(Listener listener) {
     listeners.removeListener(listener);
   }
 
+  public static String getVulkanDriverVersion(Device.Instance dev) {
+    String version = "N/A";
+    VulkanDriver vkDriver = dev.getConfiguration().getDrivers().getVulkan();
+    if (vkDriver.getPhysicalDevicesCount() > 0) {
+      version = Integer.toUnsignedString(vkDriver.getPhysicalDevices(0).getDriverVersion());
+    }
```

Comment:
It seems that the result returned here may be different from what agi actually used in the gapis? 
We directly return the version of the first vulkan physical device here, while in gapis we iterate and do compatibility checks which may actually pick other vulkan physical devices? 
If it's true, before we find a mechanism to transfer back the information of which driver we actually used, would it be better if we concat all the possible driver versions together and show them here? (If too long maybe hide with a shortened string and show in full at hover?)

Reply:
Yes, we list only the driver version of the first physical device, I assumed it was OK since the few mobile devices that we support only have one Vulkan physical device, so the trace will report only one vk physical device.

But your point is valid, I updated the PR to list the driver version of all vulkan physical devices recorded in the trace. I think it's OK to make the string long in that case, because multi-device capture would be very rare (I'm not even sure it is properly supported by AGI now).